temporal-server/rockcraft.yaml

```diff
@@ -0,0 +1,25 @@
+name: temporal-server
+# see https://documentation.ubuntu.com/rockcraft/en/1.11.0/explanation/bases/
+# for more information about bases and using 'bare' bases for chiselled rocks
+base: ubuntu@24.04 # the base environment for this rock
+version: '1.23' # just for humans. Semantic versioning is recommended
+summary: Rock for Temporal Server # 79 char long summary
+description: |
+    Temporal is a durable execution platform that enables developers to build scalable applications without sacrificing productivity or reliability. The Temporal server executes units of application logic called Workflows in a resilient manner that automatically handles intermittent failures, and retries failed operations.
+
+platforms: # the platforms this rock should be built on and run on
+    amd64:
+
```

Comment:
Is there a reason for not adding a pebble service in this rock? For example [this](https://github.com/canonical/alertmanager-rock/blob/main/0.28.0/rockcraft.yaml#L16C1-L20C21). 

I'd expect the rock to have something similar to [this](https://github.com/canonical/temporal-k8s-operator/blob/33684d34f88f6c0099176c2a5702141bbf9f4f3e/src/charm.py#L530C1-L541C15).

Reply:
For `temporal-server` to be invoked, we need to provide some configuration files for the server. I am unsure what configurations to add to the default service in the rock. I am happy to add a pebble service if we either (1) assume that the configurations will be loaded appropriately or (2) include the development config files. I am not particularly a fan of (2) since it will be superficial for the purpose of running a mock pebble service

Reply:
As per our MM discussion - I think (1) would be the best option here, accompanied by a nice piece of documentation (this rock's README), I think we can add the service. In the end, we want to replicate upstream's behaviour, which I assume to fail or wait for a configuration file to be provided. Let me know what you think.

Reply:
added pebble service for `temporal-server` in [ffb449e](https://github.com/canonical/temporal-rocks/pull/1/commits/ffb449e9e9f8d8ce8768cdb0c813142705b4990c)